src/app/workloads/workloads.component.ts

```diff
@@ -215,8 +217,8 @@ export class WorkloadsComponent implements OnInit, OnDestroy {
           this.pvc = res.pvc;
           this.pvctemp = res.pvc;
           this.pvcarray = this.pvctemp.pvc;
-          this.workloadImage = this.statefullSets[0].dockerImage;
-          this.dockerImage = this.jivaContrllers[0].openebsjivaversion;
+          // this.workloadImage = this.statefullSets[0].dockerImage;
```

Comment:
Remove the commented code from all place

Reply:
it will be required for futrue